security/pkg/nodeagent/caclient/providers/citadel/client.go

```diff
@@ -54,51 +59,40 @@ type citadelClient struct {
 	caTLSRootCert []byte
 	client        pb.IstioCertificateServiceClient
 	clusterID     string
+	conn          *grpc.ClientConn
 }
 
 // NewCitadelClient create a CA client for Citadel.
-func NewCitadelClient(endpoint string, tls bool, rootCert []byte, clusterID string) (caClientInterface.Client, error) {
+func NewCitadelClient(endpoint string, tls bool, rootCert []byte, clusterID string, isRotate bool) (caClientInterface.Client, error) {
```

Comment:
What does isRotate mean? Feel free to answer the question as a comment in the code

Reply:
Ok will delete this isRotate it seems that this flag no longer required. when first the client is created it will build connection set the isRotate to false